data_summaries/MetaMathQA.json

```diff
@@ -0,0 +1,52 @@
+{
+    "meta_math_qa": {
+        "Unique Dataset Identifier": "meta_math_qa",
+        "Dataset Name": "meta_math_qa",
+        "Paper Title": "MetaMath: Bootstrap Your Own Mathematical Questions for Large Language Models",
+        "Dataset URL": "https://huggingface.co/datasets/meta-math/MetaMathQA",
+        "GitHub URL": "https://github.com/meta-math/MetaMath",
+        "Hugging Face URL": "https://huggingface.co/datasets/meta-math/MetaMathQA",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2309.12284",
+        "Semantic Scholar Corpus ID": "262084051",
+        "Collection": "MetaMathQA",
+        "Collection URL": "https://meta-math.github.io/",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Logical and Mathematical Reasoning",
+            "Question Answering"
+        ],
+        "Text Sources": [
+            "gsm8k",
+            "math"
+        ],
+        "Model Generated": ["OpenAI GPT-3.5-Turbo"],
+        "Format": [
+            "Few-shot",
+            "Chain-of-Thought"
+        ],
+        "Human Annotation": "No",
+        "Derived from Datasets": [],
+        "Creators": [
+            "University of Cambridge",
+            "Southern University of Science and Technology",
+            "The Hong Kong University of Science and Technology",
+            "Huawei Noah’s Ark Lab",
+            "Alan Turing Institute",
+            "Max Planck Institute for Intelligent Systems"
+        ],
+        "Licenses": [
+            {
+                "License": "MIT License",
+                "License URL": "https://huggingface.co/datasets/meta-math/MetaMathQA"
+            }
+        ],
```

Comment:
Could we add the licenses for GSM8k and MATH, even though they both appear to be MIT as well?

https://github.com/openai/grade-school-math/blob/master/LICENSE
https://github.com/hendrycks/math/blob/main/LICENSE

Also, can we split "meta_math_qa" into "metamathqa_math" and "metamathqa_gsm8k"? Basically just two entries to the data summary with mostly the same attributes, but license links, text sources, dataset source will be different (and also the text metrics we calculate later). This way if anyone wants to avoid gsm8k completely for train/test overlap reasons, they can filter it out. You will also need to apply the filter in the downloader, like other collections.